core/src/com/unciv/logic/map/tile/Tile.kt

```diff
@@ -429,6 +429,40 @@ open class Tile : IsPartOfGameInfoSerialization {
         return civInfo.cities.firstOrNull { it.isWorked(this) }
     }
 
+    fun isBlockaded(): Boolean {
+        val owner = getOwner() ?: return false
+        val unit = militaryUnit
+
+        // If tile has unit
+        if (unit != null) {
+            return when {
```

Comment:
Can if/else

Reply:
Don't understand which `if/else` you are talking about here.

Reply:
My mistake, I thought the when had only 2 possible outcomes, not 3